apps/meteor/app/importer/server/methods/getImportProgress.ts

```diff
@@ -0,0 +1,38 @@
+import type { IImportProgress } from '@rocket.chat/core-typings';
+import { Meteor } from 'meteor/meteor';
+
+import { hasPermission } from '../../../authorization/server';
+import { Imports } from '../../../models/server';
+import { Importers } from '..';
+
+export const executeGetImportProgress = (): IImportProgress => {
```

Comment:
Any reason to create a new interface instead of just using the existing class as the type?

Reply:
Just to organize and also use on `packages/rest-typings/src/v1/import/import.ts`